go/ct/rlz/expressions.go

```diff
@@ -358,3 +358,34 @@ func (c constant) GetVariable() gen.Variable {
 func (c constant) BindTo(generator *gen.StateGenerator) {
 	generator.BindValue(c.GetVariable(), c.value)
 }
+
+////////////////////////////////////////////////////////////
+// Contract Account
+
+type contractAccount struct{}
+
+func ContractAccount() BindableExpression[vm.Address] {
+	return contractAccount{}
+}
+
+func (contractAccount) Domain() Domain[vm.Address] { return addressDomain{} }
+
+func (contractAccount) Eval(s *st.State) (vm.Address, error) {
+	return s.CallContext.CallerAddress, nil
+}
+
+func (contractAccount) Restrict(kind RestrictionKind, addr vm.Address, generator *gen.StateGenerator) {
+	// generator.SetPc(uint16(pc.Uint64()))
```

Comment:
Should this be code? Also applies to line 390.

Reply:
Right, this seems incomplete.